### PR TITLE
Read Only Material Order Overview & Detail Views

### PR DESCRIPTION
**Pull request by logmat**
Minimal solution to display current orders and order details.

add route material
use router link
try data table
split orders and items
improve items-per-page
improve order item list
order items per page
clean up

### CODE REVIEW

**Comment by jo-m** [comment]
@logmat @lukasheuberger  ich han eu als collaborators mit write recht zu dem repo hinzuegfüegt. D.h. ihr chönd direkt uf dem repo schaffe, anstatt uf euem eigete fork. Push to master isch restricted, aber ihr chönd en branch mache direkt i dem repo und denn en PR erstelle. Denn isches für mich eifacher zum reviewe, will ich nid euen fork mues als remote hinzuefüege bi mir lokal.

File: src/views/Material.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <v-app-bar
+      :clipped-left="$vuetify.breakpoint.lgAndUp"
+      app
+      color="blue-grey darken-3"
+      dark
+    >
+      <v-avatar :tile="true" width="140px">
```
Comment (jo-m) [inline src/views/Material.vue:9]: Could factor this out as appbar-shared-left or similar. Same for Order component.

File: src/views/Material.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <img :src="require('@/assets/logomat-logo.svg')" alt="LOGomat Logo" />
+      </v-avatar>
+      <v-toolbar-title style="width: 300px" class="ml-0 pl-4">
+        <span class="hidden-sm-and-down">LOGomat</span>
+      </v-toolbar-title>
+      <v-spacer></v-spacer>
+      <v-btn icon title="Logomat" to="/">
+        <v-icon>mdi-arrow-left</v-icon>
+      </v-btn>
+      <Settings />
```
Comment (jo-m) [inline src/views/Material.vue:22]: Could factor this out as appbar-shared-right or similar. Same for Order component.

File: src/lib/api.js [hunk 3]
```diff
@@ -81,6 +81,9 @@ export const apiAuthenticated = async (path, query = null) => {
 
   if (query !== null) {
     config.params = query;
+    config.params.limit = -1;
```
Comment (jo-m) [inline src/lib/api.js:84]: please revert this, it breaks the API for some uses (SQL error on main page).
image

**Review by jo-m — COMMENTED** [review]
please merge latest master, and run yarn pretty.

File: src/lib/api.js (hunk 3 above)
Comment (logmat) [reply src/lib/api.js]: done

**Comment by jo-m** [comment]
mysql error is still there, I think you have to revert the else branch as well, I don't think it breaks something on your side.

**Comment by logmat** [comment]
so jetzt müsste der mysql error weg sein

**Review by jo-m — APPROVED** [review]
cool, danke!

**Pull request merged**